tensorflow/compiler/xla/service/gpu/launch_dimensions.cc

```diff
@@ -101,12 +100,20 @@ StatusOr<LaunchDimensions> CalculateLaunchDimensions(
   // TODO(jlebar): Investigate this further, and tune this heuristic so we can
   // run faster on the few benchmarks where smaller block size helps.
   int64_t threads_per_block = ThreadsPerBlockLimit(gpu_device_info);
+  int64_t threads_per_block_y = 1;
```

Comment:
Could we instead combine definition and declaration? E.g. something like

```
threads_per_block_y = (cond)? CeilOfRatio(...) : 1
```

would be more readable. If `cond` gets too complex, could be extracted into a function.

Reply:
Done. I also did that for threads_per_block_x, as otherwise, it wasn't making it much clearer.